ResearchKit/Consent/ORKConsentDocument.h

```diff
@@ -148,31 +148,29 @@ ORK_CLASS_AVAILABLE
 
 /// @name PDF generation
 
-/**
```

Comment:
@rsanchezsaez Is it necessary to remove a public API? I am worried that this might cause unnecessary build failures and complications for developers who are directly calling this API..

Reply:
The thing is that `ORKHTMLPDFWriter`,  `ORKConsentSectionFormatter`, and `ORKConsentSignatureFormatter` are already internal (not even private), so I figure that if anybody is externally using this initializer, they have already modified the *ResearchKit* framework for their needs, and they'd have no problem adapting to this change.

Reply:
You're right, agreed!